set-quotas.py

```diff
@@ -122,17 +122,19 @@ def match_keystone_project(all_ks_projects, form_project):
     config.read('settings.ini')
     admin_user = config.get('auth', 'admin_user')
     admin_pwd = config.get('auth', 'admin_pwd')
-    admin_tenant = config.get('auth', 'admin_tenant')
+    admin_project = config.get('auth', 'admin_project')
+    auth_url = config.get('auth', 'auth_url')
     nova_version = config.get('nova', 'version')
-    keystone_v3_url = config.get('setpass', 'keystone_v3_url')
     quota_auth_file = config.get('quota_sheet', 'auth_file')
     quota_worksheet_key = config.get('quota_sheet', 'worksheet_key')
     quota_template = config.get('quota_email', 'template')
 
     # openstack auth
-    auth = v3.Password(auth_url=keystone_v3_url,
+    auth = v3.Password(auth_url=auth_url,
```

Comment:
Yes, we discovered that Mitaka needs the domain ids while reviewing #8 (updating toV3 auth), but I forgot to add those to the quotas PR ( #6 ) before merging it.

We need to use v3 auth with Mitaka because v2 is deprecated.  So there is no longer a need to have separate v2 and v3 auth_url's in the settings file.

When updating to v3 auth I changed all the variable names to say "project" instead of "tenant" to reflect what OpenStack is doing, and to make things less confusing.